.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
+  "parser": "babel-eslint",
+  "extends": "airbnb",
+  "globals": {
+    "__DEV__": true
+  },
+  "env": {
+    "browser": true,
+    "mocha": true
+  },
+  "rules": {
+    "no-confusing-arrow": 0,
```

Comment:
According to http://eslint.org/docs/rules/no-confusing-arrow, it seems to be a good idea to enable no-confusing-arrow. What do you think?